cachelib/cachebench/cache/Cache-inl.h

```diff
@@ -408,13 +419,20 @@ typename Cache<Allocator>::ItemHandle Cache<Allocator>::find(Key key,
     // find from cache and wait for the result to be ready.
     auto it = cache_->findImpl(key, mode);
     it.wait();
+
```

Comment:
This code looks different from https://github.com/facebook/CacheLib/blob/69738cf1fe15b42f8db374aea5483496107ffcf2/cachelib/cachebench/cache/Cache-inl.h#L405 

Are you on a stale base commit for this PR ?

Reply:
Done - rebased.